packages/hosts/base-host/src/test/host.spec.ts

```diff
@@ -0,0 +1,8 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+describe("host-placeholder", () => {
```

Comment:
This file seems not needed :)

Reply:
I removed it initially but then the build broke because there was nothing in the test folder for the test task to execute. This was the only test file here so it was either this or removing all test stuff scaffolding from the package. I don't have a strong opinion about it